sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtQueryBuilder.cs

```diff
@@ -14,15 +12,17 @@ public class AdtQueryBuilder
     {
         private readonly SelectQuery _selectQuery;
         private readonly FromQuery _fromQuery;
-        private readonly WhereQuery _whereQuery;
+        private readonly WhereStatement _whereStatement;
```

Comment:
I do like the `WhereStatement` but if the rest goes into `WhereLogic` then the SelectQuery and FromQuery also need to be `SelectLogic` and `FromLogic`. Does it seem the same to you?

Reply:
I think we can keep calling the `WhereLogic` , `WhereQuery`

Reply:
Are we distinguishing between each where segment and the fully built where clause?

Reply:
What do you mean by distinguishing? The method signatures are set up right now so that you need to have a `WhereStatement` after `FromQuery` instead of `WhereLogic` -- is that what you're thinking of?